Add details to documentation

teikmeout opened the pull request:
#180 has been merged, this PR is ready to be merged in. 

Jira Ticket: [INVENIO-196](https://jira.nyu.edu/browse/INVENIO-196)

This PR will add the following:
- [docs] symlinking Cairo packages for MacOS installations.
- [docs] need for whole Xcode as a Requirement for libxmlsec1
- [docs] brew installation at the beginning of macos steps.
- [docs] correct docker documentation link.

Please feel free to add comments on things to be changed, and reject the PR if it needs more work to be done.
Thank you for your time.

ekate commented on docs/SETUP.md:
```diff
@@ -33,28 +33,29 @@ The following instructions were modified from the [InvenioRDM System Requirement
-  - **Git**
-  - **Python Version Management (Pyenv)**
+  - **Version Control (Git 2.41.0+)**
+    - MacOS(arm) download full Xcode via the appstore (this will bring in development tools like Git, but will also setup for success when downloading Cairo). Check for installation with `xcode-select -p`. Ensure to open Xcode to accept license agreement.
```
I think you mean that full Xcode  will help to install Cairo later not just download it. Correct ?

ekate reviewed (commented):
@teikmeout I added one small comment. Please resolve and I will merge it.

Kate

teikmeout commented:
@ekate - separated xcode and homebrew installation into its own step. LMK if you like how it reads. Specified that xcode satisfies requirements for libxmlsec1 and not cairo which I had mixed up.

@lhenze - no worries. I actually was wrong since it benefits libxmlsec1 and not cairo. Corrected that last part. Added update instructions of pyenv with homebrew in case installed with that tool.

Should be good to go. LMK if you like this!
Thanks for your review.
